cwel/style/component/_icon.scss

```diff
@@ -0,0 +1,29 @@
+////
+/// @group _icon.scss
+/// @since 0.0.1
+////
+
+/// Icon fullback color is the primary
+
+svg:not(.loader) {
+    height: 20px; // stylelint-disable-line
```

Comment:
@NathanRodger It would be good to have a comment this `stylelint-disable-line`, too.